test/testdata/googleapis_test/gapic/packaging/api_defaults.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Provides default values used as metadata when constructing various packages.
+
+author: Google Inc
+description: a grpc-based api
+email: googleapis-packages@google.com
+github_user_uri: https://github.com/googleapis
+homepage: https://github.com/googleapis/googleapis
+license: Apache-2.0
+semver:
+  # TODO: Python must go to 1.0.20 or 1.1.0 when we GA, due to mistakenly
+  #       publishing some 1.0.x versions to pypi already. They are now
+  #       deleted but files still exist on the pypi servers and can prevent
+  #       us from using the version number again.
+  python:
+    version: '0.14.0'
+    next_version: '0.15.0dev'
```

Comment:
This will be done in a follow-up (there's a TODO elsewhere in the code to do this). If I do it now, it will break Python gRPC codegen. I want to submit googleapis/toolkit#870 first before I change the gRPC codegen.